src/solvers/dgsem_tree/subcell_limiters_2d.jl

```diff
@@ -27,15 +27,19 @@ end
 function (limiter::SubcellLimiterIDP)(u::AbstractArray{<:Any, 4}, semi, dg::DGSEM, t,
                                       dt;
                                       kwargs...)
-    @unpack alpha = limiter.cache.subcell_limiter_coefficients
+    (; alpha) = limiter.cache.subcell_limiter_coefficients
     alpha .= zero(eltype(alpha))
 
+    if limiter.local_minmax
+        @trixi_timeit timer() "local min/max limiting" idp_local_minmax!(alpha, limiter,
+                                                                         u, t, dt, semi)
+    end
     if limiter.positivity
         @trixi_timeit timer() "positivity" idp_positivity!(alpha, limiter, u, dt, semi)
     end
 
     # Calculate alpha1 and alpha2
-    @unpack alpha1, alpha2 = limiter.cache.subcell_limiter_coefficients
+    (; alpha1, alpha2) = limiter.cache.subcell_limiter_coefficients
```

Comment:
There's no need for this, I don't think we'll ever get rid of all the `@unpack` statements 😅

Reply:
Done in [5adf14c](https://github.com/trixi-framework/Trixi.jl/pull/1648/commits/5adf14c9d8bbb5cd043e36ad8067dfcd22880cb4)